src/util.cpp

```diff
@@ -116,7 +116,17 @@ const NamedDecl *resolveDeclaration(CompilerInstance *ci,
     // Create a new parser to handle this type parsing.
     Preprocessor &preprocessor = ci->getPreprocessor();
     Sema &sema = ci->getSema();
+#if LLVM_VERSION_AT_LEAST(7, 0, 0)
+    // TODO: Since LLVM 7, segfault occurs when the destructor of clang::Parser
+    // is called. Following code workaround the segfault by not destructing
+    // the clang::Parser instance, which leads to memory leak. This should be
+    // fixed. See https://github.com/personalrobotics/chimera/issues/222
```

Comment:
I wonder if this could be related:
https://github.com/Papierkorb/bindgen/pull/20

Reply:
Thanks for the link, but it also simply disables the code that causes the segfault (i.e., `clang::ParseAST` in that case) without fixing the root problem. :disappointed: